dockers/docker-dhcp-relay/dhcpv4-relay.agents.j2

```diff
@@ -38,3 +40,4 @@ dependent_startup_wait_for=start:exited
 
 {% endif %}
 {% endif %}
+{% endfor %}
```

Comment:
@shlomibitton have you missed a newline?

Reply:
no, this is intentionally this way, if I will add a new line it will add it in the output file also and this is not desired